src/combat/combat.js

```diff
@@ -358,6 +355,19 @@ export default class YearZeroCombat extends Combat {
     return this;
   }
 
+  /* ------------------------------------------ */
+
+  async nextTurn() {
+    const turn = this.turn;
+    const combatant = this.turns[turn];
+    const speed = combatant.getSpeedFromActor();
+    const duplicates = getCombatantsSharingToken(combatant);
+    if (speed > 1 && duplicates.length > 0) {
```

Comment:
Using the `speed > 1` means that systems that are not currently using the speed idea or when the GM duplicates the combantant manually will not remove the Slow and Fast action, no?

Reply:
I think it's OK because it's again reset at `nextRound()`.

Reply:
But if the system does not uses the speed (like Vaesen, for example), it will not reset on the second combantant, right?

Reply:
🤔 hmmm.. looks about right.

Reply:
I would recommend just to check if there is duplicates.

Reply:
That's doable.

Reply:
Updated with changes that only checks for duplicates and also retains the fast/slow actions should the combatant be the last one of the duplicates that round.